action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -135,6 +136,14 @@ func RevertLogOption() StateDBAdapterOption {
 	}
 }
 
+// CheckPutStateErrorOption set checkPutStateError as true
+func CheckPutStateErrorOption() StateDBAdapterOption {
+	return func(adapter *StateDBAdapter) error {
+		adapter.checkPutStateError = true
```

Comment:
default value should be the value used after hard fork

Reply:
i fixed. use default value false after hard-fork